tools/lsp/language/completion.rs

```diff
@@ -665,47 +667,35 @@ pub fn build_import_statements_edits(
         Position::new(map_position(&token.source_file, last.into()).line + 1, 0)
     };
 
-    for file in document_cache.documents.all_files() {
```

Comment:
Candidate for refactoring is to move this whole function (`build_import_statements_edits`) to component catalog, since it is used both by the completion and the code action, it is not really belonging only in the completion module.  But it doesn't have to be in this PR.